### DISCUSSION TITLE
Feedback for course "HTTP and curl"

### DISCUSSION THREAD

**Discussion by ludelafo** [heig-vd-dai-course/heig-vd-dai-course#121] [Polls]
Can you let us know what was easy and what was difficult for you during this chapter?

This will help us to improve the course and adapt the content to your needs. If we notice some difficulties, we will come back to you to help you.

You can use reactions to express your opinion on a comment! 🚀

Vous pouvez aussi nous laisser votre avis en français, il n'y a pas de souci !

**Comment by lucaslattion** [comment]
The "/cookie-demo" doesn't work due to the space and coma.
Working solution:
ctx.cookie("cookie", "HelloWorld!");

**Reply by ludelafo** [reply]
Thank you!

I'll have a look right now and update the course material and solution if needed.

What did you think of the new approach for the course material? The blending between the theoretical and practical content in one document?

**Reply by ludelafo** [reply]
I was able to fix the course material to reflect your remark, thank you!

A good resource I've found on the MDN Web Docs explains all the allowed (and forbidden) characters we can use in cookies: <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#cookie-namecookie-value>.

**Comment by ludelafo** [comment]
All previous comments have been marked as hidden (outdated) for the start of the new DAI semester.
